lib/web_ui/dev/integration_tests_manager.dart

```diff
@@ -4,57 +4,37 @@
 
 import 'dart:io' as io;
 import 'package:path/path.dart' as pathlib;
-import 'package:web_driver_installer/chrome_driver_installer.dart';
 
 import 'chrome_installer.dart';
+import 'driver_manager.dart';
 import 'environment.dart';
 import 'exceptions.dart';
 import 'common.dart';
 import 'utils.dart';
 
+const String unsupportedConfigurationWarning = 'WARNING: integration '
+    'tests are only supported on Chrome or on Safari(running on MacOS)';
```

Comment:
nit: "on Chrome and on Safari"
nit: add one space between "Safari" and "("

Reply:
done